scripts/elemental-airgap.sh

```diff
@@ -384,6 +418,7 @@ build_os_channel() {
         CHANNEL_IMAGE_NAME="${channel_img}-${LOCAL_REGISTRY%:*}"
     fi
     CHANNEL_IMAGE_URL="${CHANNEL_IMAGE_NAME}:${channel_tag}"
+    is_hauler && CHANNEL_IMAGE_URL=localhost:5000/${CHANNEL_IMAGE_URL}
```

Comment:
I fact I missed this too but maybe this line should be like this no?
```bash
is_hauler && CHANNEL_IMAGE_URL=${LOCAL_REGISTRY}/${CHANNEL_IMAGE_URL}
```

Reply:
no, we need to keep the localhost reference there: that's a trick, because hauler will save the image only from a registry, cannot pull it from docker. So, we start an hauler registry instance locally and push on it our just created image from docker.
Once we than get the image back from the hauler registry, hauler itself strips away the registry info, i.e., the localhost:5000.